auth.go

```diff
@@ -678,7 +694,6 @@ func (x *Central) Close() {
 	}
 	if x.Ticker != nil {
 		x.Ticker.Stop()
-		x.Ticker = nil
```

Comment:
Setting`x.Ticker = nil` is the right thing to do, and should work correctly with the simpler code flow that I mentioned in an earlier comment